scripts/readme-gen/templates/install_deps.tmpl.rst

```diff
@@ -5,14 +5,14 @@ Install Dependencies
 
     .. code-block:: bash
 
-        $ git clone https://github.com/googleapis/python-pubsub.git
+        $ git clone https://github.com/GoogleCloudPlatform/python-docs-samples.git
 
 #. Install `pip`_ and `virtualenv`_ if you do not already have them. You may want to refer to the `Python Development Environment Setup Guide`_ for Google Cloud Platform for instructions.
 
    .. _Python Development Environment Setup Guide:
        https://cloud.google.com/python/setup
 
-#. Create a virtualenv. Samples are compatible with Python 3.6+.
+#. Create a virtualenv. Samples are compatible with Python 2.7 and 3.4+.
```

Comment:
This isn't the real output, right?

Reply:
This is a template ... but it seems that synth still overrides it with old values, I'll see if these changes should be reverted. It's more important what ends up in the actual README, etc. docs.

IIRC there is already a synthtool ticket to prevent this from happening in the first place.

Reply:
Fixed that (again), and there is a [ticket](https://github.com/googleapis/synthtool/issues/663) to fix this permanently in synthtool.